### PR TITLE
feature: add the 'oproep' subsidyMeasureOfferSeries title

### PR DESCRIPTION
**Pull request by wolfderechter**
## ID
DGS-77

 ## Description

Add the 'oproep' subsidyMeasureOfferSeries title as a subtitle underneath the subsidiemaatregel.

 ## Type of change

 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
 - [ ] Other

 ## How to test

Visit the subsidiedatabank and see the newly added 'oproep' subtitles in the first column.

### CODE REVIEW

**Review by aatauil — APPROVED** [review]

File: app/components/search-table/resource/row.hbs [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <td>
   {{@consumption.subsidyMeasureOffer.title}}
-  {{#if @consumption.subsidyApplicationFlow.subsidyMeasureOfferSeries.title}}
+  {{#if @consumption.subsidyApplicationFlow.subsidyMeasureOfferSeries.label}}
     <AuHelpText @skin="secondary" class="au-u-margin-top-none">
-      ({{@consumption.subsidyApplicationFlow.subsidyMeasureOfferSeries.title}})
```
Comment (wolfderechter) [inline app/components/search-table/resource/row.hbs]: Why did you delete the `(   )`? It's in the mockup @aatauil

Comment (aatauil) [reply app/components/search-table/resource/row.hbs]: Ah sorry, found that it looked weird. you can add it back
